R/checks.R

```diff
@@ -53,3 +53,53 @@ check_reports_valid <- function(reports, model) {
     assert_numeric(reports$confirm, lower = 0)
   }
 }
+
+#' Validate probability distribution for passing to stan
+#'
+#' @description
+#' `check_stan_delay()` checks that the supplied data is a `<dist_spec>`,
+#' that it is lognormal or gamma, and that it has a finite maximum.
```

Comment:
out of date with currently supported dists.
```suggestion
#' that it is a supported distribution, and that is has a finite maximum.
```